tech/database/redis/about.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Redis
+subsection: redis
+section: tech-database
+description: In-memory data structure store, used as a database, cache and message broker
+---
+
+# Redis
+
+Redis is a free and open source in memory data structure store. It is widely used as a database, in memory cache or a message broker.
+
+## Quick start
+
+To install Redis on Fedora, run the following commands:
+
+```console
+$ sudo dnf install redis     # Install redis cli and server
+$ sudo systemctl start redis # Initialize redis server
+```
+
+To start redis on boot, run
+
+```console
+$ sudo systemctl enable redis
+```
+
+To test redis-cli, run
+
+```console
+$ redis-cli ping
+```
+
+the result from the above command will be 
+
+```console
+pong
+```
+
+## Configuring Redis
+
+The config file for Redis is located at `/etc/redis/redis.conf`.
+
+```console
+$ sudo nano /etc/redis/redis.conf
+```
+
+Redis should be always restarted after changing settings. Restart redis by running
```

Comment:
I'm just curious, isn't there `reload`?

Reply:
Restart is the suggested approach for config changes if the config file on disk is changed